idaes/models/properties/general_helmholtz/helmholtz_functions.py

```diff
@@ -20,11 +20,15 @@
 import os
 
 from matplotlib import pyplot as plt
-import numpy as np
 
 import pyomo.environ as pyo
 from pyomo.common.fileutils import find_library
 from pyomo.common.config import ConfigValue, In
+
+# importing numpy from pyomo forces proper handling of numpy types in Pyomo
+# this is a workaround for a lingering issue after Pyomo/pyomo@cfa6ff49
+# (see IDAES/idaes-pse#1348) until this is addressed in Pyomo
+from pyomo.common.dependencies import numpy as np
```

Comment:
@blnicho @jsiirola let me know if you spot any gotchas with this.

Reply:
That works.  You can also `import pyomo.common.dependencies.numpy as np` and get the same result.